src/ducks/HistoricalBalance/GetHistoricalBalance.js

```diff
@@ -48,7 +45,16 @@ class GetHistoricalBalance extends Component {
   }
 
   fetchHistoricalBalance () {
-    this.props.assets.forEach(slug => {
+    const {
+      assets,
+      wallet,
+      client,
+      interval = '1d',
+      to = new Date().toISOString(),
```

Comment:
Very cache "unfriendly". Either set it to the day/hour end, or use `getTimeIntervalFromToday`